crates/sui/src/client_commands.rs

```diff
@@ -1637,6 +1734,11 @@ impl Display for SuiClientCommandResult {
                 table.with(tabled::settings::style::BorderSpanCorrection);
                 writeln!(f, "{}", table)?;
             }
+            // todo: for all replay commands format results using tabular structure instead of
+            // todo: println statements in original command
```

Comment:
@stefan-mysten are we okay with this?
Why not polish & format first before opening it up?

Reply:
I am fine splitting this into multiple PRs to make it easier to review, thus my lack of commenting on this issue.